storage/aptosdb/src/state_merkle_db.rs

```diff
@@ -140,6 +154,33 @@ impl StateMerkleDb {
         Ok(state_merkle_db)
     }
 
+    pub(crate) fn commit(
+        &self,
+        version: Version,
+        top_levels_batch: SchemaBatch,
+        sharded_state_merkle_batches: ShardedStateMerkleSchemaBatch,
+    ) -> Result<()> {
+        TREE_COMMIT_POOL.scope(|s| {
+            let mut batches = sharded_state_merkle_batches.into_iter();
+            for shard_id in 0..NUM_STATE_SHARDS {
+                let state_merkle_batch = batches.next().unwrap();
+                s.spawn(move |_| {
+                    self.commit_single_shard(version, shard_id as u8, state_merkle_batch)
+                        .unwrap_or_else(|_| panic!("Failed to commit shard {shard_id}."));
```

Comment:
nit: equivalent to `expect()`

Reply:
linter complains it because I need format! the error string.